website/docusaurus.config.js

```diff
@@ -50,28 +50,37 @@ const config = {
         resolve: {
           modules: [resolve('node_modules'), resolve('../node_modules')],
           alias: {
-            '@deck.gl/aggregation-layers': resolve('../modules/aggregation-layers/src'),
-            '@deck.gl/arcgis': resolve('../modules/arcgis/src'),
-            '@deck.gl/carto': resolve('../modules/carto/src'),
-            '@deck.gl/core': resolve('../modules/core/src'),
-            '@deck.gl/extensions': resolve('../modules/extensions/src'),
-            '@deck.gl/geo-layers': resolve('../modules/geo-layers/src'),
-            '@deck.gl/google-maps': resolve('../modules/google-maps/src'),
-            '@deck.gl/json': resolve('../modules/json/src'),
-            '@deck.gl/layers': resolve('../modules/layers/src'),
-            '@deck.gl/mapbox': resolve('../modules/mapbox/src'),
-            '@deck.gl/mesh-layers': resolve('../modules/mesh-layers/src'),
-            '@deck.gl/react': resolve('../modules/react/src'),
-            'website-examples': resolve('../examples/website'),
+            'examples': resolve('../examples'),
+            '@deck.gl-community/bing-maps': resolve('../modules/bing-maps/src'),
+            '@deck.gl-community/graph-layers': resolve('../modules/graph-layers/src'),
+            '@deck.gl-community/react': resolve('../modules/react/src'),
+            '@deck.gl-community/layers': resolve('../modules/layers/src'),
+            '@deck.gl-community/arrow-layers': resolve('../modules/arrow-layers/src'),
+            '@deck.gl-community/editable-layers': resolve('../modules/editable-layers/src'),
+            '@deck.gl-community/react-editable-layers': resolve('../modules/react-editable-layers/src'),
+            'react-deck-graph-layers': resolve('../modules/react-graph-layers/src'),
             react: resolve('node_modules/react'),
             'react-dom': resolve('node_modules/react-dom'),
+            '@deck.gl/aggregation-layers': resolve('../node_modules/@deck.gl/aggregation-layers'),
```

Comment:
Should we make the community website buildable standalone? If we pull in checked out deck.gl during builds we may not get predictable publishing. 

We could comment these out so that they can be easily activated during development

Reply:
I'm still a quite confused about the purpose of this resolve() setup, and ocular in general, so I'd prefer if you made a pr with this relevant change.